tests/unit/routers/contacts/test_api.py

```diff
@@ -176,6 +175,17 @@ def find_default_fields(contact: ContactSchema) -> Set[str]:
     return default_fields
 
 
+SAMPLE_CONTACT_PARAMS = [
+    ("minimal_contact_data", set()),
+    ("maximal_contact_data", set()),
+    ("example_contact_data", set()),
+    ("to_add_contact_data", set()),
+    ("simple_default_contact_data", {"amo"}),
+    ("default_newsletter_contact_data", {"newsletters"}),
+    ("default_waitlist_contact_data", {"waitlists"}),
+]
+
+
 @pytest.mark.parametrize("post_contact", SAMPLE_CONTACT_PARAMS, indirect=True)
```

Comment:
Could even be put inline instead of using a constant

Reply:
True, but since it's (hopefully) being deleted very soon, I'll leave it as is in this PR.